src/main/java/it/pagopa/afm/marketplacebe/service/BundleService.java

```diff
@@ -157,12 +158,17 @@ public BundleResponse createBundle(String idPsp, BundleRequest bundleRequest) {
         LocalDateTime now = LocalDateTime.now();
         Bundle bundle = Bundle.builder()
                 .idPsp(idPsp)
+                .idChannel(bundleRequest.getIdChannel())
+                .idBrokerPsp(bundleRequest.getIdBrokerPsp())
                 .name(bundleRequest.getName())
                 .description(bundleRequest.getDescription())
                 .paymentAmount(bundleRequest.getPaymentAmount())
                 .minPaymentAmount(bundleRequest.getMinPaymentAmount())
                 .maxPaymentAmount(bundleRequest.getMaxPaymentAmount())
                 .paymentMethod(bundleRequest.getPaymentMethod())
+                .onUs(bundleRequest.getPaymentMethod().equals(PaymentMethod.CP) && CommonUtil.deNull(bundleRequest.getOnUs()))
```

Comment:
NullPointer warning: `paymentMethod` has no `@NotNull`

Reply:
It is impossible because at line 138 if `paymentMethod = null` it is set to `ANY`. The same thing for `touchpoint`.

Reply:
ok perfect